common/utils/array-utils.js

```diff
@@ -25,11 +22,7 @@ function insertIf(condition, ...elements) {
  * @returns {boolean}
  */
 function isFilledArray(potentialArray) {
-  return Boolean(
-    Array.isArray(potentialArray) &&
-      !isEmpty(potentialArray) &&
-      get(potentialArray, '[0]', undefined),
-  );
+  return Boolean(Array.isArray(potentialArray) && potentialArray.length > 0 && potentialArray[0]);
 }
 
 /**
```

Comment:
Not technically part of this PR, but we might as well fix the spelling error of `sring[]` on line 31 since it's messing up IDE type checking